spec/models/validations/financial_validations_spec.rb

```diff
@@ -713,6 +713,93 @@
         expect(record.errors["household_charge"])
           .to be_empty
       end
+
+      context "when validating ranges based on LA and needstype" do
+        before do
+          LaRentRange.find_or_create_by(
```

Comment:
Worth a Factory? Probably doesn't need to be a `find_or` since we're in a test?

Reply:
Hm, yes, so it does fail with just create, I tried truncating the table between the tests, but this is looks a bit neater methinks